apps/algolia/lib/algolia/update.ex

```diff
@@ -32,6 +32,12 @@ defmodule Algolia.Update do
     |> do_has_routes?(data)
   end
 
+  defp send_delete_and_update(update_object, base_url, index_module) do
+    with :ok <- clear_index(update_object, base_url, index_module) do
```

Comment:
question: Does it make sense to only send the update if the `clear` command is successful?

Reply:
It doesn't seem to make sense to _not_ do that. But I do think the `clear` endpoint is overkill here.